src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -207,6 +215,22 @@ public void setGradingInstruction(GradingInstruction gradingInstruction) {
         this.gradingInstruction = gradingInstruction;
     }
 
+    public List<TextAssessmentConflict> getFirst_conflicts() {
```

Comment:
Codacy found an issue: [The instance method name 'getFirst_conflicts' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968856)